com.unity.shadergraph/Editor/GraphUI/DataModel/Examples/ShaderGraphExampleTypes.cs

```diff
@@ -140,7 +140,6 @@ public object ObjectValue
     public class GraphTypeConstant : IConstant
     {
         public GraphDelta.IGraphHandler graphHandler;
-        public FieldConstant<int> lengthConstant;
```

Comment:
This was part of a larger exploration of nesting subfields in constants. In GraphType's case, we know the subfields and this code is superfluous.